assets/src/edit-story/components/colorPicker/pointer.js

```diff
@@ -42,7 +47,8 @@ const Pointer = styled.div.attrs(({ currentColor, withAlpha }) => {
   width: ${POINTER_SIZE}px;
   height: ${POINTER_SIZE}px;
   transform: translate(
-    ${({ offsetX = 0, offsetY = 0 }) => `${offsetX}px, ${offsetY}px`}
+    ${({ offsetX = 0, offsetY = 0, isRTL }) =>
+      `${(isRTL ? 1 : 1) * offsetX}px, ${offsetY}px`}
```

Comment:
```suggestion
      `${(isRTL ? -1 : 1) * offsetX}px, ${offsetY}px`}
```

Reply:
How did I manage to remove this before pushing? I clearly had this when creating the above screenshots? Weiiiiiird... 🤷‍♂️